doc/api_conf.rst

```diff
@@ -44,6 +44,11 @@ Configurable settings of the :class:`dnf.Base` object are stored into a :class:`
 
     Debug messages output level, in the range 0 to 10. Default is 2.
 
+
+  .. attribute::defaultyes
+
+    Like ``assumeyes`` but will still prompt for confirmation. The default is ``False``.
```

Comment:
Are you saying it should be added to [`options_to_move`](/rpm-software-management/dnf/blob/master/dnf/cli/option_parser.py#L64) (etc) in `options_parser.py`?